common/db_test.go

```diff
@@ -106,6 +106,13 @@ func TestDBVersion(t *testing.T) {
 	}
 }
 
+func NewStubOrch(serviceURI string, ethereumAddr string) *DBOrch {
```

Comment:
Why do we need this function? - seems like we can just use `NewDBOrch`

Reply:
True!